### DISCUSSION TITLE
EJBCA with Google CA

### DISCUSSION THREAD

**Discussion by crobinson18f** [Keyfactor/ejbca-ce#259] [General]
Hello,

Can EJBCA be configured as an RA (registration authority) for a Google Cloud Certificate Authority Service (CAS) instance?

**Comment by primetomas** [comment]
No it can not. EJBCA only works with it's own CAs. 
For an RA that talks to many other cloud CAs, you can look at products like Keyfactor Command (or similar).

**Reply by crobinson18f** [reply]
Will do. Thanks
